machado/templates/feature.html

```diff
@@ -163,6 +163,11 @@ <h4 class="card-title">
           {% for hit in data.similarity %}
             <tr>
               <td>{{ hit.program|safe }} {{ hit.programversion|safe }}</td>
+	      {% if hit.db_name == 'BLAST_SOURCE' and hit.name %}
```

Comment:
If the link works just for proteins, there should be a filter for query type polypeptide in the if statement

Reply:
added new changes assessing the request